packages/core/streams/parsers.ts

```diff
@@ -0,0 +1,53 @@
+import { AIStreamParser } from './types'
+import { trimStartOfStreamHelper } from './utils'
+
+export type AIParserPlatform = 'openai' | 'anthropic'
+
+export type AIStreamsParsersList = Record<
+  AIParserPlatform,
+  () => AIStreamParser
+>
+
+export const TEXT_PARSERS: AIStreamsParsersList = {
```

Comment:
Later, it might be good to add JSON parsers for integration with OpenAI function calling (or manual equivalents, e.g. `Respond only with valid JSON`-like prompts).